lib/chromedriver.js

```diff
@@ -226,12 +225,22 @@ class Chromedriver extends events.EventEmitter {
     } else {
       cmd = `pkill -15 -f "${this.chromedriver}.*--port=${this.proxyPort}"`;
     }
-    log.info(`Killing any old chromedrivers, running: ${cmd}`);
+    log.debug(`Killing any old chromedrivers, running: ${cmd}`);
     try {
-      await (B.promisify(cp.exec))(cmd);
-      log.info("Successfully cleaned up old chromedrivers");
+      await B.promisify(cp.exec(cmd));
+      log.debug("Successfully cleaned up old chromedrivers");
     } catch (err) {
-      log.info("No old chromedrivers seemed to exist");
+      log.warn("No old chromedrivers seemed to exist");
+    }
+    log.debug(`Clean any old adb forwarded port socket connections`);
+    let connections = await this.adb.getForwardList();
+    for (let conn of connections) {
+      if (conn.indexOf('webview_devtools') !== -1) {
+        let params = conn.split(/\s+/);
+        if (params.length > 1) {
+          await this.adb.removePortForward(params[1].replace(/[\D]*/, ''));
```

Comment:
`params[1].replace(/[\D]/g, '')` might be more stable

Reply:
It seems the params[1] can only be "tcp:port". So we strip the prefix. If ADB change the protocol in the future It should doesn't work. Because the protocol of removePortForward may also be change